ddtrace/contrib/logging/__init__.py

```diff
@@ -33,14 +38,18 @@ def hello():
 ----------------------
 
 If you prefer to instrument manually, patch the logging library then update the
-log formatter as in the following example::
+log formatter as in the following example
+
+Note that ``dd.trace_id`` and ``dd.span_id`` must come last::
 
     from ddtrace import patch_all; patch_all(logging=True)
     import logging
     from ddtrace import tracer
 
     FORMAT = ('%(asctime)s %(levelname)s [%(name)s] [%(filename)s:%(lineno)d] '
-              '[dd.trace_id=%(dd.trace_id)s dd.span_id=%(dd.span_id)s] '
+              '[dd.service=%(dd.service)s dd.env=%(dd.env)s '
+              'dd.version=%(dd.version)s] '
```

Comment:
you have an extra `]` here `dd.version=%(dd.version)s]`

Reply:
ugh dang it..

Reply:
good catch!